cocotb/share/lib/fli/FliImpl.cpp

```diff
@@ -645,7 +645,10 @@ FliIterator::FliIterator(GpiImplInterface *impl, GpiObjHdl *hdl) : GpiIterator(i
 
     LOG_DEBUG("fli_iterator::Create iterator for %s of type %d:%s", m_parent->get_fullname().c_str(), type, acc_fetch_type_str(type));
 
-    if (NULL == (selected = iterate_over.get_options(type))) {
+    try {
+        selected = &iterate_over.at(type);
+    }
+    catch (std::out_of_range const&) {
```

Comment:
I don't really think there is a reason to use exceptions here.
```suggestion
    if (iterate_over.end() == (selected = iterate_over.find(type))) {
```